internal/api/observability/forwarder.go

```diff
@@ -7,6 +7,11 @@ import (
 	"strings"
 )
 
+// NameList provides a list of names for any resource implementing Names
+type NameList interface {
```

Comment:
@jcantrill Any other suggestions for a good place to put this?  Should be somewhere in package `observability`

Reply:
I think this is fine unless we find other places where we might use. Maybe its a utility type? :man_shrugging: .  I envisioned this package as wrappers around the public API to provide useful functionality.